purchases/src/main/kotlin/com/revenuecat/purchases/PurchasesOrchestrator.kt

```diff
@@ -234,6 +234,7 @@ internal class PurchasesOrchestrator constructor(
                     this.allowSharingPlayStoreAccount,
                     appUserID,
                     marketplace = null,
+                    PostReceiptInitiationSource.RESTORE,
```

Comment:
yeah, this is a tricky one. 

For observer mode for any other store we'd do `queue`, right? Maybe we should do the same here? It'd mean we don't alias unless you specifically call logIn I suppose, which should be a good thing?

Reply:
Well, for observer mode in android we would actually do `purchase`, since it's coming from the listener. 

In Android, we only use `queue` when syncing unsynced purchases on app open/foreground. That doesn't actually use the listener.

Reply:
So for purchases outside the app, it would also be `purchase`